src/api/argo-templates/base/sat-general-iuf.template.argo.yaml

```diff
@@ -64,7 +64,7 @@ spec:
             valueFrom:
               path: "{{inputs.parameters.script_stdout_file}}"
       script:
-        image: artifactory.algol60.net/sat-docker/stable/cray-sat:3.21.2
+        image: artifactory.algol60.net/sat-docker/stable/cray-sat:csm-latest
```

Comment:
For anybody wondering, this is where the csm-latest tag gets created: https://github.com/Cray-HPE/csm/blob/b16938f51cdb10f6e09aca6107c5b33e45e19756/lib/setup-nexus.sh#L60-L63

It happens right after the cray-sat docker image is uploaded to Nexus. That `setup-nexus.sh` script gets called by `upgrade/scripts/upgrade/prerequisites.sh` from docs-csm, which is run in Stage 0.2 of the upgrade. So this will happen before the IUF is needed to start installing other products.